packages/gitbook/src/components/AIActions/AIActions.tsx

```diff
@@ -0,0 +1,188 @@
+'use client';
+
+import { useAIChatController } from '@/components/AI/useAIChat';
+import { useAIChatState } from '@/components/AI/useAIChat';
+import { ChatGPTIcon } from '@/components/AIActions/assets/ChatGPTIcon';
+import { ClaudeIcon } from '@/components/AIActions/assets/ClaudeIcon';
+import { MarkdownIcon } from '@/components/AIActions/assets/MarkdownIcon';
+import AIChatIcon from '@/components/AIChat/AIChatIcon';
+import { Button } from '@/components/primitives/Button';
+import { DropdownMenuItem } from '@/components/primitives/DropdownMenu';
+import { tString, useLanguage } from '@/intl/client';
+import { Icon, type IconName, IconStyle } from '@gitbook/icons';
+import { useState } from 'react';
+
+type AIActionType = 'button' | 'dropdown-menu-item';
+
+export function OpenDocsAssistant(props: { type: AIActionType }) {
+    const { type } = props;
+    const chatController = useAIChatController();
+    const chat = useAIChatState();
+    const language = useLanguage();
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={<AIChatIcon />}
+            label="Ask Docs Assistant"
+            description="Ask our Docs Assistant about this page"
+            onClick={() => {
+                // Open the chat if it's not already open
+                if (!chat.opened) {
+                    chatController.open();
+                }
+
+                // Send the "What is this page about?" message
+                chatController.postMessage({
+                    message: tString(language, 'ai_chat_suggested_questions_about_this_page'),
+                });
+            }}
+        />
+    );
+}
+
+export function CopyMarkdown(props: { markdown: string; type: AIActionType }) {
+    const { markdown, type } = props;
+    const [copied, setCopied] = useState(false);
+
+    // Close the dropdown menu manually after the copy button is clicked
+    const closeDropdownMenu = () => {
+        const dropdownMenu = document.querySelector('div[data-radix-popper-content-wrapper]');
+
+        // Cancel if no dropdown menu is open
+        if (!dropdownMenu) return;
+
+        // Dispatch on `document` so that the event is captured by Radix's
+        // dismissable-layer listener regardless of focus location.
+        document.dispatchEvent(new KeyboardEvent('keydown', { key: 'Escape', bubbles: true }));
+    };
+
+    return (
+        <AIActionWrapper
+            type={type}
+            icon={copied ? 'check' : 'copy'}
+            label={copied ? 'Copied!' : 'Copy for LLMs'}
+            description="Copy page as Markdown"
```

Comment:
```suggestion
            description={tString(language, 'copy_page_markdown')}
```

and then add to en.ts and other translations:
```
copy_page_markdown: 'Copy page as Markdown',
```